src/pytorch_lightning/loggers/comet.py

```diff
@@ -301,6 +301,7 @@ def experiment(self) -> Union[CometExperiment, CometExistingExperiment, CometOff
                 self._experiment = CometOfflineExperiment(
                     offline_directory=self.save_dir, project_name=self._project_name, **self._kwargs
                 )
+            self._experiment.log_other("Created from", "pytorch-lightning")
```

Comment:
Even though the behaviour from a user's perspective doesn't change at all, this should be documented in the changelog to notify the change to the community, IMO.

Reply:
Is it something from your side or I should provide the description somewhere?

Reply:
What's the best way to describe this @alexkuzmik? 

> Added:
> - The `CometLogger` now records the usage of Lightning in the experiment database

Something like that?

Reply:
@alexkuzmik If you like you can add the description directly to src/pytorch_lightning/CHANGELOG.md in the section "Added".

Reply:
@awaelchli 
"The CometLogger now flags the Comet Experiments as being created from Pytorch-Lightning for analytics purposes"

Reply:
Looks great!